README.md

```diff
@@ -19,13 +26,11 @@ Pancake
 ----------------------------------------------------------------------------------------------------------------------------------------------------------------
 
 
-## What’s inside?
+## Getting started
 
-![The Pancake tool](https://raw.githubusercontent.com/govau/pancake/master/assets/pancake.png)
+If your project is already using Pancake and you want to make changes to the outputted files you will want to change the [Pancake settings](#settings) in the `package.json` file.
 
-[Npm wrote about](http://blog.npmjs.org/post/101775448305/npm-and-front-end-packaging) the challenges frontend developers face when trying to use npm. Pancake
-is addressing those by embracing the idea of small individually versioned independent modules. Interdependencies is what npm does really well and Pancake
-will help you keep them flat and error out on conflicts. [Read more about our solution](https://medium.com/dailyjs/npm-and-the-front-end-950c79fc22ce)
+If you are creating a new project using Pancake you will want to look into creating your own [Pancake modules](#creating-your-own-pancake-modules).
```

Comment:
Getting started should be where `npm install pancake` is first shown. It should also specify that pancake installation isn't required if you are using a govau component.

Reply:
Hmm I disagree on this one, if a user has a component from the design system they shouldn't be `npm i @gov.au/pancake` this could be confusing to users.

Reply:
There needs to be a way for user to understand how to get started. They might come to this repo because they think they need to install pancake to start using DS components. So if it's not an install script maybe specify that if they are using govau components then pancake is already installed by default and they don't need to install it manually. Then the following line about editing pancake settings appeals to users that want to modify their exiting installation.